config.py

```diff
@@ -92,6 +92,80 @@ def load(self):
 The head of the HAProxy config. This contains global settings
 and defaults.
 '''))
+        self.add_template(
```

Comment:
Will returning HAProxy config itself instead of maps, when the flag is disabled work?